src/fmu/settings/_resources/pydantic_resource_manager.py

```diff
@@ -162,7 +172,12 @@ def get_model_diff(
                 f"'{incoming_model.__class__.__name__}'."
             )
 
-        IGNORED_FIELDS: Final[list[str]] = ["created_at", "created_by"]
+        IGNORED_FIELDS: Final[list[str]] = [
+            "created_at",
```

Comment:
This isn't really part of your PR either, but if moving some of this functionality to more specific managers, I think those managers should control the ignored fields too. That is, the ignore fields are defined by the child, and not a "global" list here. Perhaps as a property or class variable. @slangeveld